tests/test_uncrustify/test.py

```diff
@@ -46,12 +48,13 @@ def _diff(self, expected, actual):
         subprocess.call(cmd)
 
     # -------------------------------------------------------------------------
-    def build(self, test_input, test_lang, test_config):
+    def build(self, test_input, test_lang, test_config, test_expected):
```

Comment:
BTW, remind me why is this change needed? AFAICT this method is only used by `run_sources_tests.py` which always passes the same value. (Was this left over from an earlier version?)

Reply:
The change is for clarity. Better to have that in the caller.

Reply:
I guess that is okay, then...